great_expectations/expectations/metrics/map_metric_provider/column_condition_partial.py

```diff
@@ -166,10 +166,10 @@ def inner_func(  # noqa: PLR0913
                 metrics: Dict[str, Any],
                 runtime_configuration: dict,
             ):
-                metric_domain_kwargs = get_dbms_compatible_metric_domain_kwargs(
-                    metric_domain_kwargs=metric_domain_kwargs,
-                    batch_columns_list=metrics["table.columns"],
-                )
+                # metric_domain_kwargs = get_dbms_compatible_metric_domain_kwargs(
+                #     metric_domain_kwargs=metric_domain_kwargs,
+                #     batch_columns_list=metrics["table.columns"],
+                # )
```

Comment:
@billdirks If this doesn't cause any of our existing tests to fail can we just merge this?

I will make sure to post to `#engineering` and solicit feedback more widely before taking that step.

Reply:
Yes, if no other tests break. I think if this actually changes `metric_domain_kwargs` then, when running checkpoints, gx would always fail at metric lookup time when evaluating the expectation. Not sure if this is used in any other code path. Changing ids seems like very unexpected behavior.

Reply:
Please delete this instead of commenting out before you merge.